WebDriverAgentLib/Categories/XCUIDevice+FBHelpers.m

```diff
@@ -41,6 +62,52 @@ - (BOOL)fb_goToHomescreenWithError:(NSError **)error
   return YES;
 }
 
+- (BOOL)fb_lockScreen:(NSError **)error
+{
+  if (fb_isLocked) {
+    return YES;
+  }
+  SEL mSelector = NSSelectorFromString(@"pressLockButton");
```

Comment:
@mykola-mokhnach As you suggested, lets get rid of this by importing `#import "XCUIDevice.h"`. This method exists in the private headers

Reply:
fixed